docs/src/css/components/_dialog.scss

```
@@ -1,42 +1,125 @@
   }
 
-  button {
-    margin-top: var(--ix-space-sm);
+  [data-actions] button {
+    all: unset;
+
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+
+    padding: 0.55rem 1.25rem;
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)